README.md

```diff
@@ -20,11 +20,11 @@ To convert the DICOM dataset in a BIDS structure for this project, run the follo
 ~~~
 curate_project.py -path-in <path_DICOM_dataset> -path-output <path_BIDS_dataset>
 ~~~
-The BIDS datset is (for now) in `duke: temp/uk_biobank_BIDS`
+The BIDS datset is (for now) in **|TODO : update with raw and processed repo**
 
-Here is an example of the BIDS data structure of UK biobank data:
+Here is an example of the BIDS data structure of UK biobank dataset:
 ~~~
-uk_biobank_BIDS
+uk-biobank
```

Comment:
shouldn't it be called `uk-biobank-processed`? (ie: the repos that contains the derivatives)